packages/salmon-wallet/__test__/send.test.ts

```diff
@@ -0,0 +1,188 @@
+import waitForExpect from 'wait-for-expect'
+import { JsonRpcClient } from '@defichain/jellyfish-api-jsonrpc'
+import { GenesisKeys } from '@defichain/testcontainers'
+import { WhaleMasternodeRegTestContainer } from '@defichain/salmon-testing'
+import { SalmonWallet } from '@defichain/salmon-wallet'
+import { BigNumber } from 'bignumber.js'
+import { WhaleApiClient } from '@defichain/whale-api-client'
+import { AssetPrice } from '@defichain/salmon'
+
+const container = new WhaleMasternodeRegTestContainer()
+const address = GenesisKeys[GenesisKeys.length - 1].operator
+let client: JsonRpcClient
+let whaleApiClient: WhaleApiClient
+let oracleId: string
+
+beforeEach(async () => {
+  await container.start()
+  await container.ain.waitForWalletCoinbaseMaturity()
+
+  client = new JsonRpcClient(await container.ain.getCachedRpcUrl())
+  whaleApiClient = container.getWhaleApiClient()
+
+  await client.wallet.sendToAddress(address.address, 1)
+  await container.ain.generate(1)
+
+  oracleId = await client.oracle.appointOracle(address.address,
+    ['BTC', 'ETH', 'DOGE'].map(x => ({ token: x, currency: 'USD' })), {
+      weightage: 1.0
+    })
```

Comment:
separate this into 2 lines? easier to read that way especially when it comes to reviewing.